pyproject.toml

```diff
@@ -24,12 +24,16 @@ python = ">=3.9,<3.13"
 qibo = "^0.2.11"
 qibo_client = "^0.0.9"
 qiskit_ibm_runtime = ">=0.17"
-qiskit_ibm_provider = ">=0.8.0"
```

Comment:
are you sure that `qiskit_ibm_provider` is not needed anymore? I think I needed it for the `qiskit-client`

Reply:
It appears that `qiskit_ibm_provider` has been deprecated, see [link](quantum). I believe they have upgrade it to `qiskit_ibm_runtime`.

Reply:
Ok that's alright, as long as this does not break the `qiskit-client` I am ok with it. Just please doublecheck first, I'll do the same.

Reply:
Indeed, `qiskit_ibm_runtime` is not providing a compatible package with the same name of the old one.

You can open an issue to remind about this change, since it's due because of the deprecation. But it's not related to Braket, and so it should be decoupled from this PR.

Reply:
Thanks! Yeah. Qiskit is really confusing to use at the moment. From what I see, `qiskit_ibm_runtime` is the latest, `qiskit_ibm_provider` is deprecated. Perhaps we'll need to upgrade the Qiskit client once this done, perhaps carrying on from this PR https://github.com/qiboteam/qibo-cloud-backends/pull/22.